jablib/src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2811,13 +2811,15 @@ The\ selected\ entry\ doesn't\ have\ a\ DOI\ linked\ to\ it.=The selected entry
 Look\ up\ a\ DOI\ and\ try\ again.=Look up a DOI and try again.
 Looking\ up\ DOI...=Looking up DOI...
 No\ DOI\ found.=No DOI found.
-Cited\ By=Cited By
-Cites=Cites
+Citations\ (cited\ by)=Citations (cited by)
+References\ (cites)=References (cites)
```

Comment:
We need an ADR for that. --> new file at https://github.com/JabRef/jabref/tree/main/docs/decisions

Other developers will come with the final wording - we have some internal discussions now.

This is Option 2

Also link to semantic scholar somethow

Option 3 is the thing as it has been

Option 4

DBLP uses "References" and "Cited by"

See https://dblp.org/rec/conf/zeus/VoigtKW21.html

Option 5

Reply:
Reason: This lead to discussions and surely will lead to discussions in the future, too. And we need to ensure that the discussions to not start from zereo

The ADR needs to be linked from the code (CitationRelationsTab, line 153)